### PR TITLE
Reduce Warnings

### PR DESCRIPTION
**Pull request by Delaunay**

### CODE REVIEW

**Comment by Delaunay** [comment]
#1044

File: tox.ini [hunk 1]
```diff
@@ -261,8 +261,6 @@ commands =
 addopts = -ra -q --color=yes
 norecursedirs = .* *.egg* config docs dist build
 xfail_strict = True
-rsyncdirs = src tests
```
Comment (bouthilx) [inline tox.ini:264]: What were these for?

**Review by bouthilx — APPROVED** [review]
LGTM, Thanks!

File: tox.ini (hunk 1 above)
Comment (Delaunay) [reply tox.ini:264]: I don't know, but they trigger deprecation warnings in pytest and the messages imply the options were removed

Comment (bouthilx) [reply tox.ini:264]: Seems to be a long deprecated argument of pytest-xdist which we are not even using. :thinking: Alright, then.

**Pull request merged**